dijet/production/jet_assignment.py

```diff
@@ -59,30 +61,30 @@ def jet_assignment(self: Producer, events: ak.Array, **kwargs) -> ak.Array:
     # Check for Forward extension
     # We also use FE as a check when both jets are central
     # TODO: not implemented yet!
-    leading_is_central = np.abs(jets.eta[:, 0]) < 1.131
-    subleading_is_central = np.abs(jets.eta[:, 1]) < 1.131
-    both_central = leading_is_central & subleading_is_central
+    jet1_is_central = eta_jet1 < 1.131
+    jet2_is_central = eta_jet2 < 1.131
+    both_central = jet1_is_central & jet2_is_central
     use_fe = (
         (
-            leading_is_central |  # at least one in barrel
-            subleading_is_central
+            jet1_is_central |  # at least one in barrel
+            jet2_is_central
         ) &
         ~use_sm  # not in the same eta bin
     )
     events = set_ak_column(events, "use_fe", use_fe)
 
     # index of the central jet
-    # Use subleading since true = 1 (subleading) and false = 0 (leading)
-    central_ind = ak.values_astype(subleading_is_central, np.uint8)
+    # Use jet2 since true = 1 (subleading) and false = 0 (leading)
+    central_ind = ak.values_astype(jet2_is_central, np.uint8)
 
     # if FE, choose the central jet as the reference jet, otherwise random
     # if not SM and both central, also random
-    pro_index = ak.where(
+    ref_index = ak.where(
         use_fe & ~both_central,
         central_ind,
-        rand_ind,
+        rand_ind,  # if both central, random assignment
```

Comment:
The comment doesn't seem to cover the case where `use_fe` is `False`. I would maybe remove it since it is described in the comment above anyway.